code/game/turfs/turf.dm

```diff
@@ -257,6 +253,10 @@
 		else
 			lighting_clear_overlays()
 
+	W.levelupdate()
+	W.CalculateAdjacentTurfs()
```

Comment:
it's because CalculateAdjacentTurfs() doesn't readd it to air processing, and it is removed earlier in this proc, 235 to be specific. So it just never gets readded to LINDA processing.